### PR TITLE
Generate preview link for each PR

### LINKED ISSUE #349: Generate PR preview links
**Is your feature request related to a problem? Please describe.**
Currently, we use github preview links as PR previews. But it is hard to track the changes when there's multiple PRs from the same person. 

**Describe the solution you'd like**
1. Generate a preview link on every PR.
    Use this: https://github.com/sef-global/sef-site/blob/master/.github/workflows/main.yml 
1. Update the PR template with new URL format
   ex: https://raw.githubusercontent.com/sef-global/sef-site/master/pull_request_template.md

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
## Purpose

The purpose of this PR is to fix #349 #320

## Goals

- Generate a preview link for each Pull request.

## Approach

- Create a workflow to create a preview link for each Pull request.
- Update pull request template.

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Shall we use acf@sack.edu.lk?

We should not use the admin account for any given reason.

**Comment by Piumal1999** [comment]
@kumuditha-udayanga is this PR ready for review?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        run: jekyll build
+
+      - name: Install Surge
+        run: npm install -g surge
+
+      - name: Deploy to surge
+        env:
+          SURGE_LOGIN: acf@sack.edu.lk
+          SURGE_TOKEN: acb9d89132e778a11f8bf1fea973b9fc
+          DEPLOY_DOMAIN: https://pr-${{ github.event.number }}-sack-site-acf.surge.sh
```
Comment (Piumal1999) [inline .github/workflows/main.yml:37]: @kumuditha-udayanga can we move these values to GitHub secrets?

Comment (jayasanka-sack) [reply .github/workflows/main.yml:37]: We can't github secrets are not available for PRs from forks

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done, @kumuditha-udayanga !

**Pull request merged**